docs/sources/features/datasources/cloudmonitoring.md

```diff
@@ -335,3 +335,22 @@ Only available in Grafana v7.1+.
 
 Click on a time series in the panel to see a context menu with a link to View in Metrics Explorer in Google Cloud Console. Clicking that link opens the Metrics Explorer in the Google Cloud Console and runs the query from the Grafana panel there.
 The link navigates the user first to the Google Account Chooser and after successfully selecting an account, the user is redirected to the Metrics Explorer. The provided link is valid for any account, but it only displays the query if your account has access to the GCP project specified in the query.
+
+## Out-of-the-box dashboards
+
+> Only available in Grafana v7.3+.
+
+The updated Cloud Monitoring data source ships with pre-configured dashboards for five of the most popular GCP services:
```

Comment:
Should `data source` be `datasource` in this case? 🤔 Same for below on line 351

Reply:
Yes, per the [Word usage section of the Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#setup-set-up).

Reply:
From the [style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md#data-source):

> Two words, not one